R/stao/commercial_crew_processing_114.R

```diff
@@ -0,0 +1,35 @@
+################
+#Issue #114: Commercial Crew Member Data
+#Data Processing
+#February 2018
+#Sophia Tao
+################
+
+
+
+a <- read.csv('/home/stao/my-sasap/114_commercial_crew/Commercial Crew data 2012-2016.csv', 
+              header = T, 
+              stringsAsFactors = F, 
+              na.strings = c("", "Not Available"))
+
+# correct typos
+typo <- which(a$Full.Name == ",ARL A. HIXSON")
+a$Full.Name[typo] <- "CARL A. HIXSON"
```

Comment:
If you want, I think it's also possible to do all this in one line (without which):
`a$Full.Name[a$Full.Name == ",ARL A. HIXSON"] <- "CARL A. HIXSON"`

It does get a little bit harder to read though, so you could go either way.